tests/testthat/test-merge_sce_list.R

```diff
@@ -513,7 +513,12 @@ test_that("merging SCEs with 1 altexp and same features works as expected, with
     sce_list_with_altexp,
     batch_column = batch_column,
     # "total" should get removed
-    retain_coldata_cols = retain_coldata_cols,
+    retain_coldata_cols = c(
+      retain_coldata_cols,
+      "altexps_adt_sum",
+      "altexps_adt_detected",
+      "altexps_adt_percent"
+    ),
```

Comment:
I think for the purposes of testing simplicity it probably would have been fine to change the expected values below rather than updating this parameter, but it is fine.